src/Network/Network/Common/NetworkClient.cs

```diff
@@ -45,6 +48,7 @@ public partial class NetworkClient
         public NetworkClient(IAzureContext context)
             : this(AzureSession.Instance.ClientFactory.CreateArmClient<NetworkManagementClient>(context, AzureEnvironment.Endpoint.ResourceManager))
         {
+            SqlManagementClient = AzureSession.Instance.ClientFactory.CreateArmClient<SqlManagementClient>(context, AzureEnvironment.Endpoint.ResourceManager);
```

Comment:
This would be initialized in cmdlet that don't need it. Can you rewrite SqlManagerClient so it would be created in getter if it's null?

Reply:
Thanks for advice. Change for it for later initialization.